common/test/src/IO/NodeWriterTest.cpp

```diff
@@ -291,30 +293,30 @@ R"(// entity 0
             writer.writeMap();
 
             const std::string actual = str.str();
-            const std::string expected =
+            const std::string expected = fmt::format(
 R"(// entity 0
-{
+{{
 "classname" "worldspawn"
-}
+}}
 // entity 1
-{
+{{
 "classname" "func_group"
 "_tb_type" "_tb_layer"
 "_tb_name" "Custom Layer"
-"_tb_id" "*"
+"_tb_id" "{}"
```

Comment:
Better to check the exact value

Reply:
Hmm, I considered it, but I actually prefer to keep it as is here. My reasoning is that the point of these particular tests here is not to nail down the specific IDs, but that the node tree structure is written correctly. I'll add a specific test that nails down the IDs tho.

Reply:
see 2f0f3ce55c136372a7f02e7d5a6a9be2737a0b8c